business_rules/operators.py

```diff
@@ -304,15 +310,18 @@ def not_exists(self, other_value):
     @type_operator(FIELD_DATAFRAME)
     def equal_to(self, other_value):
         target = self.replace_prefix(other_value.get("target"))
-        comparator = self.replace_prefix(other_value.get("comparator"))
-        results = np.where(self.value.get(target) == self.value.get(comparator, comparator), True, False)
+        value_is_literal = other_value.get("value_is_literal", False)
+        comparator = self.replace_prefix(other_value.get("comparator")) if not value_is_literal else other_value.get("comparator")
+        comparison_data = self.get_comparator_data(comparator, value_is_literal)
```

Comment:
Should we combine 314-315 into a single method, since it looks like these steps are repeated throughout?

Reply:
In some cases these steps are not repeated so I left it out